.github/workflows/subscript.yml

```diff
@@ -81,22 +83,22 @@ jobs:
       - name: Update GitHub pages
         if: github.repository_owner == 'equinor' && github.ref == 'refs/heads/master' && matrix.python-version == '3.6'
```

Comment:
`matrix.python-version == '3.6'`
Should this be updated?

Reply:
That's true....Good catch. I checked the history, we used to test subscript against Python 3.6 as well. I have now updated it as well.